lib/rev-api/version.rb

```diff
@@ -1,3 +1,3 @@
 module Rev
-  VERSION = '2.2.1'
+  VERSION = '2.3.0'
```

Comment:
Minor version bump

Reply:
Here's a good guide on gem patterns. I think they are applicable to most gems/packages though! https://guides.rubygems.org/patterns/